### DISCUSSION TITLE
Support for Waydroid maybe?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [sodaliterocks/sodalite#71] [Discussion]
would it be difficult to do?

**Comment by electricduck** [comment]
As far as I can see, Waydroid is already in Fedora's main repositories. Looking at [Waydroid's docs for Fedora](https://docs.waydro.id/usage/install-on-desktops#fedora), you just need to adjust the install command slightly:

```
sudo rpm-ostree install waydroid
```

You'll need to reboot after this, or try your luck with `sudo rpm-ostree apply-live` (this applies the pending image without a reboot, but possibly inducing issues, like new kernel modules not being enabled correctly).

Once you've done that, returning to the original docs, just issue:

```
sudo systemctl enable --now waydroid-container
```

As far as I can tell, Waydroid pretty much just runs the Android runtime in a container and there shouldn't be any problems with it running on an rpm-ostree-based distro like Sodalite.

**Reply by electricduck** [reply]
Oh, having tried it myself, seems as though Waydroid **needs** Wayland &mdash; almost as if the clue is in the name 🤦 . Unfortunately, Pantheon still runs on X11 and Wayland support is still far from finished. So kind of moot at the moment 😞 .

The good news is it does seem to install properly and attempt to run (except, of course, then coming to a screaming halt when it realizes there is no Wayland session). So when Wayland arrives on Pantheon, there should be no reason it won't work!

**Reply by Heus-Sueh** [reply]
Didn't you need to install the necessary kernel modules to run waydroid? this was one of the annoying parts of installing waydroid

> Pantheon still runs on X11 and Wayland support is still far from finished. So kind of moot at the moment  .

I had forgotten about that detail lol, Thanks for the answer

**Reply by electricduck** [reply]
> Didn't you need to install the necessary kernel modules to run waydroid? this was one of the annoying parts of installing waydroid

Never tried Waydroid up until now and it just seems to... work... these days. Didn't have to do any of that.
